### PR TITLE
Add changelog and secpoll for auth-4.2.0-beta1.

### PR DESCRIPTION
**Pull request by aerique**

### CODE REVIEW

File: docs/changelog/4.2.rst [hunk 1]
```diff
@@ -5,6 +5,256 @@ Changelogs for 4.2.x
...
+    :pullreq: 7326
+    :tickets: 5430
+
+    Add ``type`` filter to search-data api.
+
+  .. change::
+    :tags: Improvements, Internals
+    :pullreq: 7502
+
+    Use a less expensive way to get memory stats for ``rel-memory-usage``.
```
Comment (pieterlexis) [inline docs/changelog/4.2.rst]: s/rel/real/

File: docs/changelog/4.2.rst [hunk 2]
```diff
@@ -5,6 +5,256 @@ Changelogs for 4.2.x
...
+
+  .. change::
+    :tags: Improvements, API
+    :pullreq: 7359
+    :tickets: 7357
+
+    Add ``rcode`` response statistics on API.
+
+  .. change::
+    :tags: Improvements, Lua
```
Comment (pieterlexis) [inline docs/changelog/4.2.rst]: Please check config.py if "Lua" is listed

File: docs/changelog/4.2.rst [hunk 3]
```diff
@@ -5,6 +5,256 @@ Changelogs for 4.2.x
...
+    :tags: Bug Fixes
+    :pullreq: 7494
+
+    Correctly interpret an empty AXFR response to an IXFR query.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 7505
+
+    Fix compilation of speedtest with GCC 8.0+.
```
Comment (pieterlexis) [inline docs/changelog/4.2.rst]: This change is not needed in the log

File: docs/changelog/4.2.rst [hunk 4]
```diff
@@ -5,6 +5,256 @@ Changelogs for 4.2.x
...
+    :pullreq: 7470
+    :tickets: 7453
+
+    Avoid duplicate NSEC3 records in presigned zones in LMDB backend.
+
+  .. change::
+    :tags: New Features, LMDB
+    :pullreq: 7453
+
+    Authortitative LMDB backend.
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]:
```suggestion
    Authoritative LMDB backend.
```

File: docs/changelog/4.2.rst [hunk 5]
```diff
@@ -5,6 +5,256 @@ Changelogs for 4.2.x
...
+    :tags: Bug Fixes, Improvements, Tools
+    :pullreq: 7364
+
+    Dnsscope off-by-one + domain-filter.
+
+  .. change::
+    :tags: Bug Fixes, Internals
+    :pullreq: 7382
+
+    Fix ``dns_random()`` always returning 0 when the minimum acceptable value is 0.
```
Comment (rgacogne) [inline docs/changelog/4.2.rst]: Wasn't this PR already included in alpha1?

Comment (aerique) [reply docs/changelog/4.2.rst]: Yes.

Comment (aerique) [reply docs/changelog/4.2.rst]: Recursor 4.2.0 Alpha 1 ;-)

**Pull request merged**